src/python/WMQuality/Emulators/DBSClient/MockDbsApi.py

```diff
@@ -88,6 +88,8 @@ def listFileLumiArray(self, **kwargs):
         if 'logical_file_name' in kwargs and len(kwargs['logical_file_name']) > 1:
             origArgs = copy.deepcopy(kwargs)
             returnDicts = []
+            if isinstance(kwargs['logical_file_name'], str):
```

Comment:
what if `logical_file_name` will not be in kwargs? I think it is better to do like `kwargs.get('logical_file_name', None)`.

Reply:
the key presence is already verified a few lines above.